packages/fiori3/src/components/AnalyticalTable/AnalyticalTable.karma.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { AnalyticalTable as Table } from './index';
```

Comment:
is there any reason why you rename the import to Table? If not I would go for `Analytical Table`.
Please also import from `../../lib/AnalyticalTable`